ext/strings.go

```diff
@@ -57,7 +57,7 @@ const (
 //	'hello'.charAt(5)  // return ''
 //	'hello'.charAt(-1) // error
 //
-// # Format
+// # Strings.Format
```

Comment:
nit: Does this need to be prefixed with a namespace? It's a receiver call on an existing string

Reply:
My mistake. Undone.